src/clr-angular/utils/drag-and-drop/providers/drag-and-drop-event-bus.spec.ts

```diff
@@ -4,50 +4,60 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import {ClrDraggableEvent} from "../interfaces/draggable-event";
-
+import {ClrDragEvent, ClrDragEventType} from "../interfaces/drag-event";
 import {ClrDragAndDropEventBus} from "./drag-and-drop-event-bus";
 
 export default function(): void {
     describe("Drag And Drop Event Bus Provider", function() {
+        let isEmitted: boolean;
+
         beforeEach(function() {
             this.dndEventBus = new ClrDragAndDropEventBus();
+            isEmitted = false;
+        });
+
+        it("should broadcast from correct Observable on drag start event type", function() {
+            this.dndEventBus.dragStarted.subscribe((event: ClrDragEvent) => {
+                isEmitted = true;
+                expect(event.type).toBe(ClrDragEventType.DRAG_START);
+            });
+            this.dndEventBus.broadcast({type: ClrDragEventType.DRAG_START});
```

Comment:
Just a heads up: these tests will not compile when you add more required properties on the `ClrDragEvent` interface. But that's fine for this PR, we'll see when we get there.

Reply:
Yup exactly, I have updated the properties here in my upcoming PR.